packages/core/primitive/src/primitive.tsx

```diff
@@ -1,3 +1,5 @@
+import * as ReactDOM from 'react-dom';
```

Comment:
I put this inside of the `core/primitive` package to colocate with the other event helper, do we feel that's the right place for it given the new deps which are react specific? would `react/primitive` be better?

Reply:
Yeah I think `react/primitive` might make more sense for that reason.